core/src/toga/app.py

```diff
@@ -483,7 +483,7 @@ def main_window(self, window):
     @property
     def current_window(self):
         """Return the currently active content window."""
-        return self._impl.current_window().interface
```

Comment:
This change will impact on any existing implementation (Cocoa / GTK).

Reply:
What does ` return self._impl.current_window().interface` return though. On windows, calling `current_window` gives out an error that it has no attribute of `interface`.

Reply:
It returns the interface level object of the window. Anywhere in the Toga codebase you see `interface`, `_impl` or `native`, it's referring to an object at one of the three levels of the API.

The reason `current_window` says it has no `interface` attribute is because, in your implementation, you're *returning* the interface level object.

Reply:
Never mind. I understood what you meant.